File: 은지/LV2/구명보트.js
function solution(people, limit) {
    var answer = 0;
    people.sort((a, b) => a-b)
    var l = 0
    var r = people.length-1
    while (l<=r) {
      if (people[l]+people[r] <= limit) {
        l += 1
        r -= 1
        answer += 1
      } else {
        r -= 1
        answer += 1
      }
    }
    return answer;
}

File: 은지/LV2/구명보트.py
def solution(people, limit):
    answer = 0
    people.sort()
    l = 0
    r = len(people)-1
    while l <= r:
        if people[l] + people[r] <= limit:
            l += 1
            r -= 1
            answer += 1
        else:
            r -= 1
            answer += 1
    return answer

p = [10, 80, 20, 40, 50]
l = 100
print(solution(p, l))